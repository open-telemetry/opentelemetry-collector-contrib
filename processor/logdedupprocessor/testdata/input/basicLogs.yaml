resourceLogs:
  - resource:
      attributes:
        - key: one
          value:
            intValue: "1"
    scopeLogs:
      - logRecords:
          - attributes:
              - key: bool
                value:
                  boolValue: true
              - key: str
                value:
                  stringValue: attr str
            body:
              stringValue: Body of the log
            severityText: info
            spanId: ""
            timeUnixNano: "1728069266547395000"
            traceId: ""
          - attributes:
              - key: bool
                value:
                  boolValue: true
              - key: str
                value:
                  stringValue: attr str
              - key: remove_me
                value:
                  boolValue: false
            body:
              stringValue: Body of the log
            severityText: info
            spanId: ""
            timeUnixNano: "1728069206547395000"
            traceId: ""
        scope: {}
