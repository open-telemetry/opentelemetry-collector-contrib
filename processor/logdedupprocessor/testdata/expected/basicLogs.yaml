resourceLogs:
  - resource:
      attributes:
        - key: one
          value:
            intValue: "1"
    scopeLogs:
      - logRecords:
          - attributes:
              - key: bool
                value:
                  boolValue: true
              - key: str
                value:
                  stringValue: attr str
              - key: log_count
                value:
                  intValue: "2"
              - key: first_observed_timestamp
                value:
                  stringValue: "2024-10-04T19:21:47Z"
              - key: last_observed_timestamp
                value:
                  stringValue: "2024-10-04T19:21:47Z"
            body:
              stringValue: Body of the log
            observedTimeUnixNano: "1728069707998122000"
            severityText: info
            spanId: ""
            timeUnixNano: "1728069708998920000"
            traceId: ""
        scope: {}
